mocked-api/handlers.ts

```diff
@@ -118,10 +118,15 @@ export const handlers = [
       ctx.status(404),
       ctx.delay(apiDelay),
       ctx.json({
-        ...error_not_found,
-        reason: `Bridge with id '${
-          bridgeId as string
-        }' for customer 'XXXXXXXX' does not exist`,
+        kind: "ErrorList",
```

Comment:
I'd have expected this to be `ErrorsResponse` not `ErrorList`.

However it is not _used_ anywhere so doesn't strictly matter.

Reply:
Sorry, I took it from a [Jira comment](https://issues.redhat.com/browse/MGDOBR-704):
![image](https://user-images.githubusercontent.com/22591802/177529967-fe727d4c-5859-4ace-a381-3265e5d54d27.png)
I'm going to update it immediately